packages/actions/test/utils/storage.ts

```diff
@@ -0,0 +1,98 @@
+import { DeleteBucketCommand, DeleteObjectCommand, S3Client } from "@aws-sdk/client-s3"
+import { fakeCeremoniesData, fakeCircuitsData } from "../data/samples"
+
+/**
+ * Deletes an object from S3 (test function only)
+ * @param bucketName <string> the name of the bucket to delete the object from
+ * @param objectKey <string> the key of the object to delete
+ * @returns <boolean> true if the object was deleted, false otherwise
+ */
+export const deleteObjectFromS3 = async (bucketName: string, objectKey: string) => {
+    if (!process.env.AWS_ACCESS_KEY_ID || !process.env.AWS_SECRET_ACCESS_KEY)
+        // throw new Error("Missing AWS credentials, please add them in the .env file")
+        return false
+    const s3 = new S3Client({
```

Comment:
nit: make an utility to get a new S3Client to avoid code repetition among methods.